apps/api/src/app/events/usecases/create-notification-jobs/create-notification-jobs.command.ts

```diff
@@ -22,6 +22,9 @@ export class CreateNotificationJobsCommand extends EnvironmentWithUserCommand {
   @IsDefined()
   template: NotificationTemplateEntity;
 
+  @IsDefined()
+  templateProviderIds: Map<ChannelTypeEnum, string>;
```

Comment:
there is nothing related to the template, so I feel it should be called `channelToProviderIds`?